mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/annotations/Dao.java

```diff
@@ -56,4 +56,13 @@
  */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.CLASS)
-public @interface Dao {}
+public @interface Dao {
+
+  /**
+   * When the new instance of a class annotated with {@code @Dao} is created an automatic check for
+   * schema validation is performed. It verifies if all {@code @Dao} entity fields are present in
+   * CQL table. If not the exception is thrown. This check has startup overhead so once your app is
+   * stable you may want to disable it. The schema Validation check is enabled by default.
+   */
+  boolean enableEntitySchemaValidation() default true;
```

Comment:
>The mapper builder could have an option to turn this off (for faster startup once the app is stable).

To see that property working please take a look at the test that using this `@Dao`:
https://github.com/datastax/java-driver/pull/1245/files#diff-af276dd4cc66aa56a5f9c0dcf2d8d611R180

The implementation is here:
https://github.com/datastax/java-driver/pull/1245/files#diff-1d89fc55d9834b473028aaba93c44dc9R255